lib/Byng/Pimcore/ElasticSearch/Filter/TagKeyFilter.php

```diff
@@ -1,101 +1,104 @@
 <?php
+
 /**
- * TagKeyFilter.php
- * Definition of class TagKeyFilter
- * 
- * Created 16-Mar-2015 16:00:55
+ * This file is part of the "byng/pimcore-elasticsearch-plugin" project.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the LICENSE is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  *
- * @author M.D.Ward <matthew.ward@byng.co>
- * @copyright (c) 2015, Byng Services Ltd
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
  */
-namespace ElasticSearch\Filter;
-
 
+namespace Byng\Pimcore\Elasticsearch\Filter;
 
 /**
  * TagKeyFilter
- * 
- * @author M.D.Ward <matthew.ward@byng.co>
+ *
+ * @author Elliot Wright <elliot@byng.co>
+ * @author Matt Ward <matt@byng.co>
  */
-class TagKeyFilter implements FilterInterface
+final class TagKeyFilter implements FilterInterface
 {
-    
     /**
-     *
      * @var array
      */
-    protected $replaceable = [' ', '-', '\''];
-    
+    private $replaceable = [ " ", "-", "\"" ];
+
     /**
-     *
      * @var string
      */
-    protected $replacement = '_';
-    
-    
-    
+    private $replacement = "_";
+
+
     /**
-     * 
-     * @param array $replaceable
-     * @param type $replacement
+     * Constructor
+     *
+     * @param array|null $replaceable
+     * @param string|null $replacement
      */
     public function __construct(array $replaceable = null, $replacement = null)
```

Comment:
`$replaceable` could do with being defaulted to an empty array actually, so the null check wouldn't be necessary.